daemon/daemon.go

```diff
@@ -885,6 +885,17 @@ func NewDaemon(c *Config) (*Daemon, error) {
 			config.alwaysAllowLocalhost = true
 		}
 	}
+	// At this point, k8s.Init() allocated the IPv4AllocPrefix and the
```

Comment:
Lines 888-889 vs. 891-892 seem redundant to me. Remove one of the two sets of lines.

Reply:
I've reordered the sentences